src/main/java/io/jenkins/update_center/HealthScores.java

```diff
@@ -0,0 +1,54 @@
+package io.jenkins.update_center;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.alibaba.fastjson.JSON;
+import io.jenkins.update_center.util.HttpHelper;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+
+public class HealthScores {
+    private static final String HEALTH_SCORES_URL = "https://reports.jenkins.io/plugin-health-scoring/scores.json";
```

Comment:
Rather than using the application endpoint, which generate the output on demand, this stores a static file, generated every hour IIRC.

Reply:
True per https://github.com/jenkins-infra/infra-reports/blob/main/plugin-health-scoring/Jenkinsfile#L1